src/notation/qml/MuseScore/NotationScene/internal/PartialTiePopup.qml

```diff
@@ -0,0 +1,115 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2022 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Window 2.15
+
+import Muse.Ui 1.0
+import Muse.UiComponents 1.0
+import MuseScore.NotationScene 1.0
+
+StyledPopupView {
+    id: root
+    margins: 0
+
+    property NavigationSection notationViewNavigationSection: null
+    property int navigationOrderStart: 0
+    property int navigationOrderEnd: partialTieNavPanel.order
+
+    property QtObject model: partialTiePopupModel
+
+    showArrow: false
+
+    onOpened: {
+        tieMenuLoader.show(Qt.point(0, 0))
```

Comment:
It is a little bit of an ugly hack to have a StyledPopupView with nothing in it, and then launch a ContextMenuLoader from it. It would be nicer if we can avoid the StyledPopupView. That would require a bit of not-entirely-trivial refactoring though in, `ElementPopupLoader`. It might be better to save that for a next PR. Also, we should check if this hack does not cause graphical weirdnesses, like an extra border somewhere, or double drop shadow in a corner; if it does, that might increase the priority of the follow-up refactoring PR a bit.

Reply:
Yeah, I agree it's messy and not ideal.  I'd like to address it outside of this PR though.